### PR TITLE
Button and badge

### PR DESCRIPTION
**Pull request by ArpitPatni**
Hi guys
Could you please review my button and badge component.

### CODE REVIEW

File: Styles/badge.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  align-items: flex-end;
+  justify-content: center;
+  margin: 25px;
+}
+.icon-avatar-badge {
+  position: absolute;
+  bottom: 0.5rem;
+  right: 0rem;
+  background-color: var(--green-color);
+  color: var(--white-color);
```
Comment (sakshi006) [inline Styles/badge.css:33]: there is some repetitive code, maybe you can create a common class for them and include the repetitive styles in that.

**Review by sakshi006 — COMMENTED** [review]
Good to go.

**Review by harshitpaliwal95 — COMMENTED** [review]
looks good

File: Styles/badge.css (hunk 1 above)
Comment (ArpitPatni) [reply Styles/badge.css:33]: Thank you for the review!

**Pull request merged**